benchmarks/cwe_mitre.toml

```diff
@@ -0,0 +1,25 @@
+[general]
+    # benchmark name
+    name = 'CWE'
+    # select the most applicable category from the list provided by this repo
+    category = 'CODING_STANDARD'
+    # mark your benchmark with custom tags (your own custom tags OR use existing from other projects)
+    tags = ['portability', 'security', 'safety']
+
+[info]
+    # programming language
+    language = 'language agnostic'
```

Comment:
I you sure, I thought that they have some different packages with different languages

Reply:
CWE is a general list of information security rules. Most of the rules are a general safety recommendation. But there are also a small number of rules aimed at recommendations specifically for the C, C++. Java and PHP languages.